tools/niminst/niminst.nim

```diff
@@ -144,23 +146,30 @@ const
 Usage:
   niminst [options] command[;command2...] ini-file[.ini] [compile_options]
 Command:
-  csource             build C source code for source based installations
-  scripts             build install and deinstall scripts
-  zip                 build the ZIP file
-  inno                build the Inno Setup installer
-  nsis                build the NSIS Setup installer
-  deb                 create files for debhelper
+  csource                build C source code for source based installations
+  scripts                build install and deinstall scripts
+  zip                    build the ZIP file
+  inno                   build the Inno Setup installer
+  nsis                   build the NSIS Setup installer
+  deb                    create files for debhelper
 Options:
-  -o, --output:dir    set the output directory
-  -m, --main:file     set the main nim file, by default ini-file with .nim
-                      extension
-  --var:name=value    set the value of a variable
-  -h, --help          shows this help
-  -v, --version       shows the version
+  -o, --output:dir       set the output directory
+  -m, --main:file        set the main nim file, by default ini-file with .nim
+                         extension
+  --var:name=value       set the value of a variable
+  --withLinenoise:on|off whether to include linenoise.h, on by default
```

Comment:
1. It should be ``off`` by default.
2. The switch should be more general like ``--bundleNim`` for Nim compiler specific logic.